backend/src/model/entity.rs

```diff
@@ -20,214 +20,583 @@ use super::r#enum::{
+
+    /// - The list of the common names of the plant in German.
+    /// - *Fetched from* Wikidata API if not present in any source datasets.
+    /// - *Fill ratio:* 25%
     pub common_name_de: Option<Vec<Option<String>>>,
-    /// Family of the plant.
+
+    /// - Family of the plant.
+    /// - See `/doc/architecture/glossary.md`.
+    /// - *Used* to build up hierarchy.
+    /// - *Fetched from* PracticalPlants and Permapeople.
+    /// - *Fill ratio:* 88%
     pub family: Option<String>,
-    /// Genus of the plant.
+
+    /// - Genus of the plant.
+    /// - Is also first word of unique_name, see above (`/doc/database/hierarchy.md`)
+    /// - *See also* `/doc/architecture/glossary.md`.
+    /// - *Used* to build up hierarchy.
+    /// - *Fetched from* PracticalPlants and Permapeople.
+    /// - *TODO:* copy from first word of unique name.
+    /// - *Fill ratio:* 63%
     pub genus: Option<String>,
-    /// Fetched from Permapeople as \`edible_uses\` and merged with Reinsaat.  (fetched from Permapeople and Reinsaat)
+
+    /// - The edible use of the plant, answering: Which food can be produced from this plant?
+    /// - Interesting for search functionality.
+    /// - *Fetched from* Permapeople as `edible_uses` and merged with Reinsaat.
+    /// - *Fill ratio:* 6%
     pub edible_uses_en: Option<String>,
-    /// Fetched from PracticalPlants as \`medicinal_uses\` and merged with Permapeople. (fetched from PracticalPlants and Permapeople)
+
+    /// - Not used.
+    /// - *Fetched from* PracticalPlants as `medicinal_uses` and merged with Permapeople.
+    /// - *Fill ratio:* 1%
     pub medicinal_uses: Option<String>,
-    /// (fetched from PracticalPlants)
+
+    /// - Only for references.
+    /// - *Fetched from* PracticalPlants)
+    /// - *Fill ratio:* 34%
     pub material_uses_and_functions: Option<String>,
-    /// (fetched from PracticalPlants)
+
+    /// - Only for references.
+    /// - *Fetched from* PracticalPlants)
+    /// - *Fill ratio:* 63%
     pub botanic: Option<String>,
-    /// (fetched from PracticalPlants)
+
+    /// - Only informational.
+    /// - *Fetched from* PracticalPlants
+    /// - *Fill ratio:* 1%
     pub material_uses: Option<String>,
-    /// (fetched from PracticalPlants)
+
+    /// - *Used* for search ranking (diversity).
+    /// - *Fetched from* PracticalPlants)
+    /// - *Fill ratio:* 13%
     pub functions: Option<String>,
-    /// (fetched from PracticalPlants)
+
+    /// - Not used.
+    /// - *Use* hardiness_zone instead.
+    /// - *Fetched from* PracticalPlants
+    /// - *Fill ratio:* 0.05%
     pub heat_zone: Option<i16>,
-    /// (fetched from PracticalPlants)
+
+    /// - Shade tolerance of the plant, to be used together with light_requirement.
+    /// - *Used* in shade layer.
+    /// - *For example* a plant that has "no shade", should get a warning if placed in a shade.
+    /// - *Fetched from* PracticalPlants.
+    /// - *Fill ratio:* 63%
```

Comment:
Shade indicates the shade tolerance. Plants can grow at better light conditions, however.
Warnings should only show if the a plant is moved into a too dark spot. No warning when moved into a lighter spot.